mtags/src/main/scala-3/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -148,13 +148,21 @@ class CompletionProvider(
     import MemberOrdering._
     var relevance = 0
     val sym = completion.value.sym
+
+    def hasGetter = try {
+      // isField returns true for some classes
+      sym.isField || sym.getter != NoSymbol
```

Comment:
`sym.isField` returns true for StringIndexOutOfBoundsException which is causing some of the other ordering issues. I will raise a PR in Dotty.

Reply:
Does it return true for `StringIndexOutOfBoundsException` class or object?

There is a weird thing that classpath search might produce object symbols that don't actually exist.
I've just got an idea how to fix it and will try that.

Reply:
Och, i haven't actually checked. Maybe that's the reason?